package.json

```diff
@@ -30,40 +30,31 @@
     "node": ">=4.8.4"
   },
   "scripts": {
-    "test": "grunt test"
+    "clean": "rm -rf package-lock.json && rm -rf node_modules && rm -rf coverage",
+    "test": "mocha --recursive 'test/**/*.js' --reporter spec --timeout 3000 --ui bdd --exit",
+    "test:watch": "npm run test -- -w ./lib",
+    "lint": "jshint lib/ --config .jshintrc && jshint test/ --config test/.jshintrc",
+    "test:coverage": "istanbul cover _mocha -- --recursive 'test/**/*.js' --reporter spec --exit",
+    "watch": "watch 'npm test && npm run lint' ./lib ./test"
   },
   "dependencies": {
-    "async": "1.5.2",
-    "request": "~2.69.0",
-    "underscore": "1.8.3",
-    "cheerio": "0.20.0",
-    "xmldom": "0.1.22",
-    "logops": "1.0.0-alpha.7",
+    "async": "2.6.1",
+    "request": "~2.88.0",
+    "underscore": "1.9.1",
+    "cheerio": "1.0.0-rc.2",
+    "xmldom": "0.1.27",
+    "logops": "2.0.0",
     "iotagent-node-lib": "git://github.com/telefonicaid/iotagent-node-lib.git#master",
-    "lwm2m-node-lib": "git://github.com/telefonicaid/lwm2m-node-lib.git#master"
+    "lwm2m-node-lib": "git://github.com/dcalvoalonso/lwm2m-node-lib.git#task/updateDependencies"
   },
   "devDependencies": {
-    "nock": "0.48.0",
-    "should": "8.2.2",
-    "grunt-contrib-jshint": "0.12.0",
-    "grunt-contrib-watch": "0.6.1",
-    "grunt-gjslint": "0.2.1",
-    "grunt": "0.4.5",
-    "mocha": "2.4.5",
-    "mongodb": "2.2.35",
-    "grunt-mocha-test": "0.12.7",
-    "chai": "~1.8.0",
-    "sinon": "~1.7.3",
-    "istanbul": "~0.1.34",
-    "grunt-exec": "0.4.6",
-    "grunt-dox": "0.5.0",
-    "grunt-github-pages": "0.0.4",
-    "grunt-plato": "~1.3.0",
-    "sinon-chai": "2.4.0",
-    "proxyquire": "0.5.1",
-    "grunt-mkdir": "0.1.2",
-    "grunt-contrib-clean": "0.7.0",
-    "grunt-githooks": "0.5.0",
-    "closure-linter-wrapper": "1.0.1"
+    "istanbul": "~0.4.5",
+    "jshint": "~2.9.6",
+    "mocha": "5.2.0",
+    "mongodb": "3.1.8",
```

Comment:
As it is explained in https://github.com/telefonicaid/lwm2m-node-lib/pull/136#discussion_r232181093, the IOTA is functional working against replica sets. Mongodb is a development dependency in order to run the unit tests because as you can see in https://github.com/telefonicaid/lightweightm2m-iotagent/blob/master/test/unit/active-attributes-test.js#54 and https://github.com/telefonicaid/lightweightm2m-iotagent/blob/master/test/unit/active-attributes-test.js#75, after each test the corresponding MongoDB databases are cleaned.

Reply:
Fixed with a2aa33b15d82f281492532e2b19ccb9737c8b2c6